main.go

```diff
@@ -41,7 +41,7 @@ func main() {
 	writer := tabwriter.NewWriter(os.Stderr, 0, 8, 0, '\t', 0)
 
 	// Read the config file.
-	var configFile = flag.String("f", localbase+"./paladin.conf-test",
+	var configFile = flag.String("f", localbase+"/etc/paladin.conf",
```

Comment:
This change should not appear in either of these commits. If I merge this, it will hide the proper history of this line.  Please remove it.